cyrw/device.go

```diff
@@ -67,6 +67,8 @@ func hex32(u uint32) string {
 }
 
 func (d *Device) Init(cfg Config) (err error) {
+	d.lock()
```

Comment:
Yes, good, move lock up to driver entry points.  That's better.  We'll need to remember to set lock when we add other entry points like d.SetMACAddr()